### DISCUSSION TITLE
reuse "trained" voice

### DISCUSSION THREAD

**Discussion by alafortu** [SWivid/F5-TTS#109] [Ideas]
I'm a complete newbie to this, but was wondering if it would be possible to save the "model" of the voice generated to be able to reuse it. A new tab could be created "load voice" where we could load the "trained voice" and input new text to generate an audio of that text with that voice without having to feed the audio sample again. I don't know if that even makes sense or if it would create an efficiency... maybe not. :)

**Comment by SWivid** [comment]
I may not have quite got your point. Here is what I understand you might want to express:
1. Want to use the reference audio seen by the model during training -> thought it's just fine to use in a zero-shot way (i.e. feel free to upload any voice and make a try)
2. Want to get the same result for multiple runs with same input -> https://github.com/SWivid/F5-TTS/blob/ab78265b4d6e4c512374baaea56d7c46a35cb61b/inference-cli.py#L224-L232
fix and pass in e.g. `seed=0` (need to manually do re-rank, or modified to random generated a seed number and record it)
